packages/nimbus/src/components/calendar/calendar.mdx

```diff
@@ -22,14 +22,14 @@ a single date.
 ## Overview
 
 The calendar component is most often used within the popover component when a
-user interacts with a date input. Easily toggle between month, year, and single day selection.
+user interacts with a date input. Easily toggle between month, year, and single
+day selection.
 
 ### Resources
 
 Deep dive on details and access design library.
 
-[Figma Link](https://www.figma.com/design/AvtPX6g7OGGCRvNlatGOIY/NIMBUS-design-system?m=auto&node-id=5818-15651)
-[Storybook Link](https://nimbus-storybook.vercel.app/?path=/docs/components-date-calendar--docs)
```

Comment:
Removed since we don't link Storybook elsewhere

Reply:
@stephsprinkle funny timing, i just mentioned this on @jaikamat 's  PR. i thought we were going to start doing this, @ellelevy?
if so, ill create a task for myself to add them all.

Reply:
yeah, I thought linking storybook was fine and that what we decided was to not _embed_ storybook?